src/main/java/ch/wisv/areafiftylan/utils/db/migration/V20170507210556__rename_username_to_email.java

```diff
@@ -0,0 +1,11 @@
+package ch.wisv.areafiftylan.utils.db.migration;
+
+import org.flywaydb.core.api.migration.spring.SpringJdbcMigration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+public class V20170507210556__rename_username_to_email implements SpringJdbcMigration {
```

Comment:
Let me check, but as far as I remember these tables operate with the `Id` in the `Users` table and not the `Username` or `Email`.